neonvm/runner-image-loader/daemonset.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: neonvm-runner-image-loader
+  namespace: neonvm-system
+spec:
+  selector:
+    matchLabels:
+      name: neonvm-runner-image-loader
+  template:
+    metadata:
+      labels:
+        name: neonvm-runner-image-loader
+    spec:
+      containers:
+      - image: runner:dev
+        name: neonvm-runner-loader
+        command: ["sh", "-c", "echo 'image loaded and container started' && sleep 100d"]
```

Comment:
What happens after 100d? Container exits and is restarted by the DaemonSet?

Reply:
That's what would happen, yeah - if we somehow didn't deploy for 100d :sweat_smile: